src/bake/DevServer.zig

```diff
@@ -9,7 +9,7 @@
 pub const DevServer = @This();
 pub const debug = bun.Output.Scoped(.Bake, false);
 pub const igLog = bun.Output.scoped(.IncrementalGraph, false);
-
+const Chunk = @import("../bundler//bundle_v2.zig").Chunk;
```

Comment:
avoid file imports if possible

const Chunk = bun.bundle_v2.Chunk

(move this at the end of the file)